Offseason2022/src/main/java/frc/robot/commands/AutoWait.java

```diff
@@ -3,19 +3,24 @@
 
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.subsystems.Drivetrain;
 
 /**
  *
  */
 public class AutoWait extends WaitCommand
 {
-  private double m_timeout;
+  private int        m_timerNum;
+  private Drivetrain m_drivetrain;
+  private double     m_waitTime;
 
-  public AutoWait(double timeout)
+  public AutoWait(Drivetrain drivetrain, int timerNum)
   {
-    super(timeout);
-    m_timeout = timeout;
+    super(timerNum);

```

Comment:
I don't think we want to have the AutoWait command reference the drivetrain subsystem. It doesn't use any of it's resources. AutoWait should just be a command we can use to put a delay in between other commands.

Also, I don't know why "super" was used by robot builder to generate the template, but we should not need that either. 

When the command is used with "new", we should pass a timer number in, and that timer number is used to select between two different wait times.